--- ./src/extras/xmlformatter.c.orig	2006-02-03 18:24:47.000000000 +0100
+++ ./src/extras/xmlformatter.c	2006-02-03 18:25:03.000000000 +0100
@@ -8,9 +8,9 @@
 
 #include <string.h>
 #include <gtk/gtk.h>
-#include <SAX.h>
-#include <parser.h>
-#include <parserInternals.h>
+#include <libxml/SAX.h>
+#include <libxml/parser.h>
+#include <libxml/parserInternals.h>
 
 #include <unistd.h>
 
--- ./src/xmlparser.c.orig	2006-02-03 18:24:23.000000000 +0100
+++ ./src/xmlparser.c	2006-02-03 18:24:39.000000000 +0100
@@ -9,9 +9,9 @@
 #include <string.h>
 #include <gtk/gtk.h>
 #ifndef WIN32
-#include <SAX.h>
-#include <parser.h>
-#include <parserInternals.h>
+#include <libxml/SAX.h>
+#include <libxml/parser.h>
+#include <libxml/parserInternals.h>
 #else
 #include <libxml/SAX.h>
 #include <libxml/parser.h>
--- ./src/xmlview-control.c.orig	2001-11-20 03:25:48.000000000 +0100
+++ ./src/xmlview-control.c	2006-11-02 23:23:15.000000000 +0100
@@ -11,8 +11,8 @@
 #ifdef ENABLE_BONOBO
 
 #include "xmlview-control.h"
-#include <parser.h>
-#include <parserInternals.h>
+#include <libxml/parser.h>
+#include <libxml/parserInternals.h>
 
 #include "xmlparser.h"
 #include "xmlwindow.h"
